libs/shared/src/components/UX/Input/index.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { createRef, useEffect, useState } from 'react';
+import { Form, Label, Input, Popup } from 'semantic-ui-react';
+import { FormattedMessage, MessageDescriptor, useIntl } from 'react-intl';
+import { BaseProps } from '../types';
+import { LessVars } from "@thecointech/site-semantic-theme/variables";
+import { MessageWithValues } from '../../../types';
+
+const isMessage = (messageOrComponent: BaseProps["intlLabel"]): messageOrComponent is MessageDescriptor =>
+  messageOrComponent.hasOwnProperty("defaultMessage")
+
+export const UxInput = (props: BaseProps) => {
```

Comment:
Git seems to have lost this files history!  There was a bit of rework here, but not that much!

Reply:
The major change is that the client used to be responsible for supplying valid messages.

Now UxInput calls the validator, and then displays the error message (if one is supplied)